vic_run/src/calc_surf_energy_bal.c

```diff
@@ -192,16 +192,12 @@ calc_surf_energy_bal(double             Le,
         while (soil_con->Zsum_node[i] < soil_con->depth[0]) {
             i++;
         }
-        T1_old =
-            energy->T[i -
-                      1] +
-            (energy->T[i] -
-             energy->T[i -
-                       1]) *
-            (soil_con->depth[0] -
-             soil_con->Zsum_node[i -
-                                 1]) /
-            (soil_con->Zsum_node[i] - soil_con->Zsum_node[i - 1]);                                                                                                        // i should be greater than 0, else code would have aborted in initialize_model_state()
+        T1_old = energy->T[i - 1] +
+                 (energy->T[i] - energy->T[i - 1]) *
+                 (soil_con->depth[0] -
+                  soil_con->Zsum_node[i -
+                                      1]) /
```

Comment:
In that case it'll happen every time. We should see if we can change this in the uncrustify settings or whether we can exclude white space only changes from commits.

> On Jan 13, 2015, at 9:42 AM, Joe Hamman <notifications@github.com> wrote:
> 
> In vic_run/src/calc_surf_energy_bal.c:
> 
> > @@ -192,16 +192,12 @@ calc_surf_energy_bal(double             Le,
> >          while (soil_con->Zsum_node[i] < soil_con->depth[0]) {
> >              i++;
> >          }
> > -        T1_old =
> > -            energy->T[i -
> > -                      1] +
> > -            (energy->T[i] -
> > -             energy->T[i -
> > -                       1]) *
> > -            (soil_con->depth[0] -
> > -             soil_con->Zsum_node[i -
> > -                                 1]) /
> > -            (soil_con->Zsum_node[i] - soil_con->Zsum_node[i - 1]);                                                                                                        // i should be greater than 0, else code would have aborted in initialize_model_state()
> > +        T1_old = energy->T[i - 1] +
> > +                 (energy->T[i] - energy->T[i - 1]) *
> > +                 (soil_con->depth[0] -
> > +                  soil_con->Zsum_node[i -
> > +                                      1]) /
> 
> done. (I've been fixing these as I go through the code but this is a uncrustify thing.
> 
> —
> Reply to this email directly or view it on GitHub.
>